gino/dialects/base.py

```diff
@@ -67,18 +67,58 @@ def raw_transaction(self):
 
 
 class PreparedStatement:
-    def __init__(self):
+    def __init__(self, clause=None):
         self.context = None
+        self.clause = clause
 
     def iterate(self, *params, **kwargs):
         return _PreparedIterableCursor(self, params, kwargs)
 
+    async def _do_execute(self, multiparams, params, one=False,
+                          return_model=True):
+        ctx = self.context.connection.execute(
+            self.clause, *multiparams, **params).context
+        if ctx.executemany:
+            raise ValueError('PreparedStatement does not support multiple '
+                             'parameters.')
+        assert ctx.statement == self.context.statement, 'Please report a bug'
+        params = []
+        for val in ctx.parameters[0]:
+            if asyncio.iscoroutine(val):
+                val = await val
```

Comment:
this should be taken care at call sites?

Reply:
Urh yeah you're probably right about it. It was a necessary `await` for `_ResultProxy.execute()` because it is a workaround for the schema generator. But no one should have used prepared statement yet, so I'll remove this one.